### PR TITLE
Add Test case for EggDropping

### PR DESCRIPTION
**Pull request by naanagon**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Comment by naanagon** [comment]
@siriak , @yanglbme Test cases passed. Can you please take a look..?

**Comment by naanagon** [comment]
@siriak , @yanglbme - could you please review, build is green now.

**Review by yanglbme — APPROVED** [review]

File: src/test/java/com/thealgorithms/dynamicprogramming/EgsDroppingTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.thealgorithms.dynamicprogramming;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EgsDroppingTest {
```
Comment (yanglbme) [inline src/test/java/com/thealgorithms/dynamicprogramming/EgsDroppingTest.java:7]:
```suggestion
public class EggDroppingTest {
```

please rename the class and java file

**Review by yanglbme — CHANGES_REQUESTED** [review]

File: src/test/java/com/thealgorithms/dynamicprogramming/EgsDroppingTest.java (hunk 1 above)
Comment (naanagon) [reply src/test/java/com/thealgorithms/dynamicprogramming/EgsDroppingTest.java]: Done

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Pull request merged**